portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +9,101 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div>
+        <h1>Sofia Vega</h1>
+        <div>
+          <!--Shows a picture of me-->
+          <div id="crop-circle">
+            <img src="images/foto.jpg" id="my-photo">
+          </div>
+          <div>
+            <!--This div show icons for my github, linkedin and cv links-->
+            <div id="links">
+              <a href="https://github.com/SofiaVega">
+                <img src="images/github.png" height="32" width="32">
```

Comment:
Use alt text for images: https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback